app/models/vm_reconfigure_task.rb

```diff
@@ -29,7 +29,7 @@ def self.get_description(req_obj)
     unless req_obj.options[:vm_memory].blank?
       new_settings << "Memory: #{req_obj.options[:vm_memory].to_i} MB"
     end
-    new_settings << "Processor Sockets #{req_obj.options[:number_of_cpus].to_i}" unless req_obj.options[:number_of_cpus].blank?
+    new_settings << "Processor Sockets #{req_obj.options[:number_of_sockets].to_i}" unless req_obj.options[:number_of_cpus].blank?
```

Comment:
@lfu I am always seeing a "Processor Sockets" count of 0 in the request UI.  Tried on both VMware and RHEV.
![image](https://cloud.githubusercontent.com/assets/2591569/10175088/138a9dde-66bb-11e5-91e5-d00d53be631d.png)